boms/geode-all-bom/src/test/resources/expected-pom.xml

```diff
@@ -470,27 +470,27 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-annotations</artifactId>
-        <version>2.13.3</version>
+        <version>2.17.0</version>
       </dependency>
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-core</artifactId>
-        <version>2.13.3</version>
+        <version>2.17.0</version>
       </dependency>
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-databind</artifactId>
-        <version>2.13.3</version>
+        <version>2.17.0</version>
       </dependency>
       <dependency>
         <groupId>com.fasterxml.jackson.datatype</groupId>
```

Comment:
https://mvnrepository.com/artifact/com.fasterxml.jackson/jackson-bom Jackson provides a corresponding BOM file, so I think replacing it directly would be better.